src/careamics/models/lvae/noise_models.py

```diff
@@ -34,18 +34,67 @@ def noise_model_factory(
         If the chosen noise model `model_type` is not implemented.
         Currently only `GaussianMixtureNoiseModel` is implemented.
     """
-    if model_config:
```

Comment:
We need to check if `model_config` exists. Indeed, when we instantiate for example `muSplit` stuff in the `lightining_module` we don't need the noise model, and hence we will pass a `None` `model_config`